src/Algebra/Lattice.hs

```diff
@@ -126,7 +126,7 @@ meetLeq x y = (x /\ y) == x
 -- > Absorption: a \/ (a /\ b) == a /\ (a \/ b) == a
 class (JoinSemiLattice a, MeetSemiLattice a) => Lattice a where
 
--- | A join-semilattice with some element |bottom| that \/ approaches.
+-- | A join-semilattice with an identity element 'bottom' for '\/'.
```

Comment:
The identity statement is definitely correct, but so is the previous one. The more elements you `join` together the *closer* you'll get to the `bottom`. E.g. in

![image](https://user-images.githubusercontent.com/51087/32714478-c2bf606a-c856-11e7-9859-4d505d22d435.png)

if you join *VP_0* and *VP_1*  (lower right part) you get *VP*, which is closer to *⊥*.

---

If you can have both notions in the haddock, it would be great. The latter could make the idea of/intuition for lattices click for someone.

---

*Note* I suspect that previous haddock is copied from somewhere, but I don't know the reference.

Reply:
Should their join not be V? I would think VP is their meet.

Reply:
Me = fool. Indeed, `a \/ b` is approaching `top`, not `bottom`. Merging this as is.